__tests__/app.test.js

```diff
@@ -69,5 +69,46 @@ describe("App API, /api", () => {
           });
       });
     });
+    describe.only("GET request with article_id parametric endpoint", () => {
+      it("returns a status of 200 when successfully reached", () => {
+        return request(app).get("/api/articles/1").expect(200);
+      });
```

Comment:
This test is unnecessary, as you will be checking for a 200 status code on line 79 😄